package.json

```diff
@@ -29,12 +29,16 @@
   },
   "homepage": "https://github.com/mapbox/dynamodb-replicator",
   "dependencies": {
+    "@aws-sdk/client-cloudwatch": "^3.445.0",
```

Comment:
When you do `npm ls aws-sdk` do you still see it anywhere? Not saying that if you did it would block this PR but I don't think we are done until we remove it at all levels